libtransmission/clients.c

```diff
@@ -234,6 +234,12 @@ char* tr_clientForId(char* buf, size_t buflen, void const* id_in)
             tr_snprintf(buf, buflen, "\xc2\xb5Torrent Embedded %d.%d.%d%s", strint(id + 3, 1), strint(id + 4, 1),
                 strint(id + 5, 1), getMnemonicEnd(id[6]));
         }
+        else if (strncmp(chid + 1, "UW", 2) == 0)
+        {
+            tr_snprintf(buf, buflen, "\xc2\xb5Torrent Web %d.%d.%d%s", strint(id + 3, 1), strint(id + 4, 1), strint(id + 5,
```

Comment:
As a general rule I don't like to move code that's otherwise untouched to avoid polluting commits. This entire file should probably have the parsing logic generalized with the clients kept in a config file as it's quite ungainly to manage.

Reply:
Wouldn't you just be adding code, the same way you added uTorrent Web here?

Reply:
I can merge once this requested change is made.